graphql/resolvers.go

```diff
@@ -1937,6 +1937,9 @@ func (r *mutationResolver) AddAnnotationIssue(ctx context.Context, taskID string
 	apiIssue restModel.APIIssueLink, isIssue bool) (bool, error) {
 	usr := MustHaveUser(ctx)
 	issue := restModel.APIIssueLinkToService(apiIssue)
+	if err := annotations.ValidateIssueURL(issue.URL); err != nil {
+		return false, InternalServerError.Send(ctx, fmt.Sprintf("issue does not have valid URL: %s", err.Error()))
```

Comment:
this seems more like a [InputValidationError](https://github.com/evergreen-ci/evergreen/blob/822f3633bfa1e9f8c95d49f91c602e0b65bb7072/graphql/errors.go#L21)